### PR TITLE
add PYTH to prices table

### PR DESCRIPTION
**Pull request by cctdaniel**
# Thank you for contributing to Spellbook!
Please refer to the top of the `readme` in the root of Spellbook to learn how to contribute to Spellbook on DuneSQL.

### CODE REVIEW

**Comment by cctdaniel** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by cctdaniel** [comment]
recheck

File: models/prices/solana/prices_solana_tokens.sql [hunk 1]
```diff
@@ -42,6 +42,7 @@ FROM
     ('sol-solana', 'solana', 'SOL', 'So11111111111111111111111111111111111111112', 9),
     ('ust-terrausd', 'solana', 'USTC', '9vMJfxuKxXBoEa7rM12mYLMwTacLMLDJqHozw96WQL8i', 6),
     ('usdc-usd-coin', 'solana', 'USDC', 'EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v', 6),
-    ('usdt-tether', 'solana', 'USDT', 'Es9vMFrzaCERmJfrF4H2FYD4KCoNkY11McCe8BenwNYB', 6)
+    ('usdt-tether', 'solana', 'USDT', 'Es9vMFrzaCERmJfrF4H2FYD4KCoNkY11McCe8BenwNYB', 6),
+    ('pyth-pyth-network', 'solana', 'PYTH', 'HZ1JovNiVvGrGNiiYvEozEVgZ58xaU3RKwX8eACQBCt3', 6)
```
Comment (Hosuke) [inline models/prices/solana/prices_solana_tokens.sql:46]: API: https://api.coinpaprika.com/v1/coins/pyth-pyth-network

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @cctdaniel

**Comment by andrewhong5297** [comment]
@aalan3 @jeff-dude can we merge this?

**Pull request merged**